Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM debian:buster-slim
+
+RUN set -x \
+  `# workaround for error in debian-slim during openjdk installation` \
+  && mkdir -p /usr/share/man/man1 \
+  && apt-get update \
+  && apt-get install --no-install-recommends -y curl git gnupg openjdk-11-jre-headless python3 python3-pip \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN set -x \
+  && echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list \
+  && curl -fsL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | apt-key add \
+  && apt-get update \
+  && apt-get install -y sbt \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN set -x \
+  && git clone https://github.com/VirtuslabRnD/scalapy.git \
+  && cd scalapy \
+  `# VirtuslabRnD:kwargs as of 27-05-2020` \
+  && git checkout 2597f7b23301ddd59067de24d4d3809cf475ae9f \
+  && sed -i 's/^scalaVersion in ThisBuild := scala213Version$/scalaVersion in ThisBuild := scala212Version; version in ThisBuild ~= (_.replaceFirst("\\\\+[^+]+$", ""))/' build.sbt \
+  && sbt publishLocal
+
+
+RUN set -x \
+  && git clone https://github.com/Avasil/scalapy-numpy.git \
+  && cd scalapy-numpy \
+  && git checkout shape-reshape \
+  && sed -i 's/^scalaVersion in ThisBuild := scala213Version$/scalaVersion in ThisBuild := scala212Version; version in ThisBuild ~= (_.replaceFirst("\\\\+[^+]+$", ""))/' build.sbt \
+# build using scalapy published earlier
```

Comment:
This can possibly be done less intrusively by just excluding transitive dep from a released version of scalapy-numpy (`0.1.0+6-14ca0424`) to the version of `scalapy-core` it points to + explicitly depending on scalapy-core `0.3.0+31-94930a4d` from our project